fasthttpproxy/example_test.go

```diff
@@ -0,0 +1,112 @@
+package fasthttpproxy
+
+import (
+	"log"
+	"time"
+
+	"github.com/valyala/fasthttp"
+	"golang.org/x/net/http/httpproxy"
+)
+
+func ExampleDialer_GetDialFunc() {
+	p := Dialer{
+		TCPDialer: fasthttp.TCPDialer{
+			Concurrency: 1000,
+		},
+		Timeout:        time.Second * 30,
+		ConnectTimeout: time.Second * 20,
+		Config: httpproxy.Config{
+			HTTPProxy:  "http://127.0.0.1:8005",
+			HTTPSProxy: "http://127.0.0.1:8005",
+		},
+		DialDualStack: true,
+	}
+	// proxy information comes from environment
+	dialer, err := p.GetDialFunc(false)
```

Comment:
Shouldn't this be `true` if you want the config to come from the env?

Reply:
My thinking was somewhat flawed; I should have combined the three examples together. Thank you.